app/src/main/java/com/gsandchase/mileagelogger/MileageCounter.java

```diff
@@ -0,0 +1,70 @@
+package com.gsandchase.mileagelogger;
```

Comment:
I will implement those eventually, but for now I'd just like a working app for weekly mileage. I also don't know what serializing a file is.